ShejaEddy/Politico
# #171643967 user cast vote

####What does this PR do?
It helps users cast their vote to a given politician
#### Description of Task to be completed?
- create all offices pages
- user chooses an office to vote in
- redirects to candidates in that office
#### How should this be manually tested?
After cloning my repo
- checkout ft-user-cast vote
- and run home.html
- the u should be seeing a vote option
#### What are the relevant pivotal tracker stories?
#171643967
